sass/widgets/menu/_menu_item.scss

```diff
@@ -13,6 +13,15 @@
             }
         }
 
+        > div {
+            padding-left: 0px !important;
+            &:hover,
+            &:focus {
+                background-color: var(--center-channel-color-10) !important;
```

Comment:
For _submenu_item.css it was needed as the mobile submenu is a modal that is overriding its defaults pretty heavily.   
For _menu_item.css I honestly don't remember why I did that.  I believe it was to override the div/a/button styling for menu items.

I am open to trying to do it without `!important` for both, however honestly scss is not my strongest skill and it may take a bit.

Reply:
I propose leaving the css as is unless further recommendation on approach is offered.